jobs/keepalived/templates/bin/utils/openstack/generate_token

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#sourcing job properties to bring openstack credential in env variables
```

Comment:
Any reason for not adding set -e and pipefail?

Reply:
will add.

Reply:
https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/pull/93#discussion_r197063704